rpcs3/rpcs3qt/gl_gs_frame.cpp

```diff
@@ -100,14 +107,18 @@ void gl_gs_frame::set_current(draw_context_t ctx)
 		{
 			fmt::throw_exception("Could not bind OpenGL context");
 		}
-	}
+	});
 }
 
 void gl_gs_frame::delete_context(draw_context_t ctx)
 {
 	const auto gl_ctx = static_cast<GLContext*>(ctx);
 
-	gl_ctx->handle->doneCurrent();
+	// Calling this on the main thread is necessery in debug builds.
```

Comment:
Same here, cannot be done from main thread. Anything with "current" is thread-local